assembly/std/arguments.ts

```diff
@@ -16,7 +16,7 @@ import {encode, decode} from './base64';
  *
  */
 export class Args {
-  private offset: i64 = 0;
+  private offset: i32 = 0;
```

Comment:
Is there a specific reason to change this variable from an `i64` to `i32` ?

Reply:
because the length of a string is a i32: https://www.assemblyscript.org/stdlib/string.html#fields